### ISSUE TITLE
PhoneGap does not stop loading

### ISSUE THREAD

**Issue by MyGIS** [phonegap/phonegap-app-desktop#913]
After installation, PhoneGap does not stop loading when trying to opening it for the first time.

**Comment by johnhemming** [comment]
I have exactly the same issue.

One possible cause may be that I initially loaded the PhoneGap build CLI using npm and then I ran the windows installer for the desktop application.

**Comment by rgilley** [comment]
Same issue for me, fresh install of phonegap on windows 7 64bit. First time ever to install this, but doesnt work

**Comment by johnhemming** [comment]
At least that demonstrates that it is not because I have previously installed cordova or the phonegap CLI.  I am using Windows 10 64bit.

**Comment by johnhemming** [comment]
I have been rummaging around on the adobe website and it may be that Adobe are not really offering this service any more. (or at least not to new users).

**Comment by 360webfirm2020** [comment]
I am having the same issue. It just keeps on loading and never stops. I installeed Node, and cordova and tried again, no go. Would really like to use this program if I can ever get it to work. 

Anybody ?

**Comment by johnhemming** [comment]
It could be someone key being on holiday.  This is a thread on the adobe community forum that asks some questions.
https://community.adobe.com/t5/phonegap/the-death-of-phonegap-real-or-not/td-p/10813752

**Comment by johnhemming** [comment]
It is also possible that some servers are switched off so the desktop cannot log in.

**Comment by chrisgriffith** [comment]
My recommendation is not to use this app and develop directly with the CLI. Adobe is no longer supporting the PhoneGap effort, so no one is actively working on this app. It is an open-source project, so you could fork it and build it yourself...

**Comment by drago1231** [comment]
Was having the same issue with Windows 10 (x64), but I got it working by doing the following:

1. Installed the desktop app using the win32 binary (0.4.5 beta)
2. Downloaded the node.js version 6 x86 zip (https://nodejs.org/dist/latest-v6.x/node-v6.17.1-win-x86.zip)
3. In the zip file, went into node_modules folder, extracted the npm folder, and zipped the npm folder into npm.zip
4. Navigated to the 'C:\Program Files (x86)\Adobe\PhoneGap\PhoneGap Desktop\resources\app.asar.unpacked\bin' folder, and replaced npm.zip with the one I created.
5. Opened cmd, navigated to the same folder, and ran postInstall.bat

In a nutshell, PhoneGap comes with npm v3.10.3, and this will replace that with npm v3.10.10, which includes many bug fixes for Windows. Hope this helps!

**Comment by MyGIS** [comment]
Thank you master @drago1231  you have solved my problem. I can now run PhoneGap but did not test it fully.

**Comment by johnhemming** [comment]
Thanks for this, but because of the uncertainty relating to PhoneGap I decided to work with basic Cordova and having got things to work at that level I am going to ignore PhoneGap.  There is an adobe community thread about this.

https://community.adobe.com/t5/phonegap/can-new-users-sign-up-to-use-phonegap-either-in-the-cloud-or-on-the-desktop/m-p/10839735

**Comment by drago1231** [comment]
@johnhemming @chrisgriffith having now played around with the desktop app, i agree with just using the cordova cli. the desktop app doesn't seem to add any value over the cli

**Comment by gtmsingh** [comment]
@drago1231 Thanks for the mitigation steps. It worked for me too for version 0.4.5

**Comment by akshay-s-770** [comment]
Thank you master @drago1231 you have solved my problem.

**Comment by KeitaroCL** [comment]
Thank you @drago1231 , i can solve this!! two months trying... youre the best!

**Comment by carlosdecolonia** [comment]
> 
> 
> Was having the same issue with Windows 10 (x64), but I got it working by doing the following:
> 
>     1. Installed the desktop app using the win32 binary (0.4.5 beta)
> 
>     2. Downloaded the node.js version 6 x86 zip (https://nodejs.org/dist/latest-v6.x/node-v6.17.1-win-x86.zip)
> 
>     3. In the zip file, went into node_modules folder, extracted the npm folder, and zipped the npm folder into npm.zip
> 
>     4. Navigated to the 'C:\Program Files (x86)\Adobe\PhoneGap\PhoneGap Desktop\resources\app.asar.unpacked\bin' folder, and replaced npm.zip with the one I created.
> 
>     5. Opened cmd, navigated to the same folder, and ran postInstall.bat
> 
> 
> In a nutshell, PhoneGap comes with npm v3.10.3, and this will replace that with npm v3.10.10, which includes many bug fixes for Windows. Hope this helps!

Thank you. Works for me!

**Comment by ragcsalo** [comment]
Just an additional info: running `postInstall.bat` from the CLI resulted in an ERROR message:

> Error: EPERM: operation not permitted, chmod 'C:\ProgramData\Adobe\PhoneGap\npm\.mailmap'

Solution: I had to right click `postInstall.bat` and **run it as an Administrator**. After that PhoneGap Desktop started normally. :-)

**Comment by howardkhl** [comment]
64bit version worked for me.
https://github.com/phonegap/phonegap-app-desktop/releases/download/0.1.7/PhoneGapSetup-win64.exe

Refer to this thread
https://github.com/phonegap/phonegap-app-desktop/issues/739

**Comment by JesseAntoni** [comment]
> 64bit version worked for me.
> https://github.com/phonegap/phonegap-app-desktop/releases/download/0.1.7/PhoneGapSetup-win64.exe
> 
> Refer to this thread
> #739

>how do you guys look at the application that your are making with phonegap

**Comment by rorijo** [comment]
Thank you @drago1231 you have solved my problem.

**Comment by Myyummy** [comment]
Thanks @JesseAntoni !! Now I can use it properly!!
